org.blackfennec.app.yml

```diff
@@ -11,15 +11,16 @@ modules:
   - name: blackfennec
     buildsystem: simple
     build-commands:
-      - cp -r ./ /app/
       - install -Dm644 ./org.blackfennec.app.desktop /app/share/applications/org.blackfennec.app.desktop
       - install -Dm644 ./icon.png /app/share/icons/hicolor/128x128/apps/org.blackfennec.app.png
       - install -Dm644 ./org.blackfennec.app.metainfo.xml /app/share/metainfo/org.blackfennec.app.metainfo.xml
-      - pip3 install /app
+      - install -Dm644 ./org.blackfennec.app.metainfo.xml /app/org.blackfennec.app.metainfo.xml
```

Comment:
why do you install this there?

Reply:
because we use the metainfo.xml as the source for info in e.g. the about window. But when we pip install the package it does not get copied... thus we need to explicitly make it available in the flatpak.